src/pages/api/events/[eventId]/present/[volId].ts

```diff
@@ -1,2 +1,37 @@
-export {};
+import { NextApiRequest, NextApiResponse } from "next";
+import { markVolunteerPresent } from "server/actions/Volunteer";
+import errors from "utils/errors";
+import { APIError } from "utils/types";
+
 // GET /api/events/[eventId]/present/[volId] will mark volunteer volId as present for event eventId
```

Comment:
Can you change this to POST? I know I put GET before but GET requests aren't supposed to change the state of the server at all.

Reply:
hahaha I was thinking about that but I had blind faith in you lol